tests/Behat/features/element-editor.feature

```diff
@@ -20,11 +20,6 @@ Feature: View types of elements in an area on a page
       And I should see "Bob's Block" as the title for block 2
       And I should see "Some content II" as the summary for block 2
 
-    Examples:
-      | group  |
-      | ADMIN  |
-      | AUTHOR |
```

Comment:
The scenario outline wasn't actually using these variables. See [the docs on scenario outlines](https://docs.behat.org/en/v2.5/guides/1.gherkin.html#scenario-outlines)
There's no reason to use admin here anyway so it makes more sense to just remove the examples and change the outline to a normal scenario.